agar-mini-map.user.js

```diff
@@ -95,7 +95,14 @@ window.msgpack = this.msgpack;
                         }
                     }
                     mini_map_party.trigger('update-list');
-
+                    break;
+                case 130:
+                    if (agar_server != packet.data.url) {
+                        var region_name = $("#region>option[value='"+packet.data.region+"']").text();
+                        var gamemode_name = $("#gamemode>option[value='"+packet.data.gamemode+"']").text();
+                        alert("game server mismatched\n\n" + "Your game server is:\n" + agar_server
+                            + "\n\nMini-map game server is:\n" + packet.data.url + "\n" + region_name + ", " + gamemode_name + packet.data.party);
```

Comment:
@domen111 I was thinking that there might be some problems when a tab freezed. A player might be just in a game and get killed because of a freeze. If you don't have time to do that, I will modify this after I merge this PR.